src/electionguard/api/create_election.c

```diff
@@ -134,7 +136,7 @@ void API_CreateElection_free(struct joint_public_key joint_key,
 {
     for (uint32_t i = 0; i < api_config.num_trustees; i++)
     {
-        if (trustee_states[i].bytes != NULL)
+        if (trustee_states != NULL && trustee_states[i].bytes != NULL)
```

Comment:
For efficiency, the check for trustee_states != NULL should be done outside the for loop.